openedx/core/djangoapps/content_staging/tests/test_clipboard.py

```diff
@@ -155,11 +154,11 @@ def test_copy_html(self):
         assert olx_response.get("Content-Type") == "application/vnd.openedx.xblock.v1.html+xml"
         # For HTML, we really want to be sure that the OLX is serialized in this exact format (using CDATA), so we check
         # the actual string directly rather than using assertXmlEqual():
-        assert olx_response.content.decode() == dedent("""
-            <html url_name="toyhtml" display_name="Text"><![CDATA[
-            <a href='/static/handouts/sample_handout.txt'>Sample</a>
-            ]]></html>
-        """).lstrip()
+        assert olx_response.content.decode() == (
+            "<html url_name=\"toyhtml\" display_name=\"Text\"><![CDATA["
+            "<a href='/static/handouts/sample_handout.txt'>Sample</a>"
+            "]]></html>\n"
+        )
```

Comment:
```suggestion
        assert olx_response.content.decode() == dedent("""
            <html url_name="toyhtml" display_name="Text"><![CDATA[
            <a href='/static/handouts/sample_handout.txt'>Sample</a>
            ]]></html>
        """).replace("\n", "")
```
What do you think about this? I find it much more readable without the `\` escapes everywhere.

Reply:
👍🏻